ci/build-test.sh

```diff
@@ -6,3 +6,5 @@ cd "$(dirname "$0")/../"
 
 ci/build.sh
```

Comment:
should we remove this file and directly call ci/codegen.sh? It looks like the changes in both of our .NET PRs make `ci/build.sh` completely obsolete.